lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -123,26 +123,11 @@
 
 desc 'Build puppet module package'
 task :build do
-  Rake::Task['build:pdk'].invoke
-end
-
-namespace :build do
-  desc 'Build Puppet module with PDK'
-  task :pdk do
-    require 'pdk/util'
-    require 'pdk/module/build'
-
-    path = PDK::Module::Build.invoke(force: true, 'target-dir': File.join(Dir.pwd, 'pkg'))
-    puts "Module built: #{path}"
-  rescue LoadError
-    _ = `pdk --version`
-    unless $CHILD_STATUS.success?
-      warn 'Unable to build module. Please install PDK or add the `pdk` gem to your Gemfile.'
-      abort
-    end
-
-    system('pdk build --force')
-  end
+  require 'puppet/modulebuilder'
+  target_dir = File.join(Dir.pwd, 'pkg')
+  builder = Puppet::Modulebuilder::Builder.new(Dir.pwd, target_dir, nil)
```

Comment:
Logger already defaults to `nil`
```suggestion
  builder = Puppet::Modulebuilder::Builder.new(Dir.pwd, target_dir)
```

And `target_dir` defaults to `File.join(source, 'pkg')` so you could even simplify it to:
```suggestion
  builder = Puppet::Modulebuilder::Builder.new(Dir.pwd)
```

Reply:
Thanks @ekohl, I've updated!